transcom/mymove
# INT-B-20477

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A980122&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Created a new endpoint to cancel moves.

### How to test

1. Navigate to http://officelocal:3000/swagger-ui/ghc.html#/move/moveCancellation
2. Grab a move id from the db and execute the endpoint
3. Check database that move status is now cancelled
